sage-ios/SAGE/SAGE/UIView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  UIView.swift
+//  SAGE
+//
+//  Created by Andrew on 9/30/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    
+    func setX(x: CGFloat) {
+        var frame = self.frame
+        frame.origin.x = x
+        self.frame = frame
```

Comment:
nitpicky but for all these methods do you need to set self.frame = frame?
won't the object change so you don't need to do this